content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -267,3 +280,53 @@ $ kubectl delete -f $HOME/istio-remote.yaml
 {{< text bash >}}
 $ helm delete --purge istio-remote
 {{< /text >}}
+
+## Stable access to Istio control plane services
+In the above procedure, endpoint IPs of Istio services are gathered and used to
+invoke Helm that will create Istio services on the remote clusters. This updates
+kube-dns in the remote clusters so it can be used to resolve the Istio service names.
```

Comment:
Does kube-dns in remote cluster updated? How it was updated?

Reply:
I can add a bit more clarity on this